tasks/managed/push-artifacts-to-cdn/tests/test-push-artifacts-to-cdn-fail-ir-failure.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-artifacts-to-cdn-fail-ir-failure
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: |
+    Run the push-artifacts task with the InternalRequest failing and ensure the task fails. The InternalRequest
+    itself will never fail, but the status.results field of it will reflect the failure.
```

Comment:
See my other comment - we even have a test for this - so the IR should never fail, so you probably don't want to change the internal task to return the error code.

Reply:
yes. I reverted this and re-worked it so that internal-pipelineruns can be used too